manifests/odh/odh-admins.yaml

```diff
@@ -3,4 +3,4 @@ kind: Group
 metadata:
   name: odh-admins
 users:
-  - user1
+  - "b64:kube:admin"
```

Comment:
@goern What does this change mean?

Reply:
this generally enables the kube-admin user to be a odh admin.